core/src/main/java/io/javaoperatorsdk/webhook/admission/validation/DefaultAdmissionRequestValidator.java

```diff
@@ -25,8 +26,9 @@ public DefaultAdmissionRequestValidator(Validator<T> validator) {
   public AdmissionResponse handle(AdmissionRequest admissionRequest) {
     var operation = Operation.valueOf(admissionRequest.getOperation());
     var originalResource = (T) getTargetResource(admissionRequest, operation);
```

Comment:
Do I understand properly that if its a DELETE operation then both old and originalResource will be present and will be the same (the old resource)?

Reply:
Yes correct. I think it would simplify implementation to avoid checking null value and conditional on operation check.

Reply:
Hmm, not sure if this would create more confusion. At least something I instantly caught on to.

I think AdmissionRequest should be leading for the terminology, so if we receive a Delete request, and only the oldResource is filled in the AdmissionRequest it is a bit confusing that we will the old and the target request. So would rather go with filling only the oldResource  in this case. But if you have different view / rational we can discuss.

Reply:
Sure. Fine for me. I will also change the behavior of the getTargetResource. Passing directly the result of the admission request is better.

Will fix tomorrow